src/main/java/cc/hyperium/gui/keybinds/GuiKeybinds.java

```diff
@@ -174,6 +173,13 @@ private boolean exceedsBoundaries(int yPosition) {
         return partitions;
     }
 
+    @Override
+    public void show() {
+        initialGuiScale = Minecraft.getMinecraft().gameSettings.guiScale;
+        Minecraft.getMinecraft().gameSettings.guiScale = 2;
+        super.show();
```

Comment:
What is this method's purpose?

Reply:
To reset the GUI scale back to what it was previously for GUIs that only work with a specific GUI scale. This is kind of a band aid fix and should be resolved at some point in the future.

Reply:
Ah I see.  Maybe we should create a common method or class for it.